src/status_im/ui/screens/browser/events.cljs

```diff
@@ -73,7 +73,9 @@
  :open-url-in-browser
  [re-frame/trim-v]
  (fn [cofx [url]]
-   (let [browser {:browser-id    (random/id)
+   (let [browser {:browser-id    (or (http/url-host url)
```

Comment:
i can see in testing "No scheme" returns nil, in that case better to apply `http/normalize-and-decode-url` before, move it to `let` and use normalized url in `browser-id` and in `history`, thanks

Reply:
Not quite sure what is wanted, but it sounds like the desired change is to thread the url thru normalize-and-decode-url before url-host. Makes sense, but plz confirm.

Reply:
Or I could move it inside url-host, but I think it better url-host stay lower level and offer just what goog.uri does, leave the caller to decide if they want to normalize beforehand.

Reply:
yes, to thread the url thru normalize-and-decode-url before url-host. but we have it already in history below, so it should be something like this
```
(let [normalized-url (http/normalize-and-decode-url dapp-url)
        browser {:browser-id   (or (http/url-host  normalized-url) (random/id))
                  :name          name
                  :dapp?         true
                  :history-index 0
                  :history       [normalized-url]}]
     (model/update-browser-and-navigate cofx browser))))
```
thanks

Reply:
Haha, tunnel vision on my part. Got it.